TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/datahandlers/JsonAccumulatorTest.java

```diff
@@ -50,8 +39,14 @@ private static Object readJson(byte[] testFileBytes, int chunkBound) throws IOEx
 
     byte[] getData(String key) throws IOException {
         switch (key) {
-            case TOY:
+            case TINY:
                 return "{\"name\":\"John\", \"age\":30}".getBytes(StandardCharsets.UTF_8);
+            case MEDIUM:
+                return randomJsonGenerator.getRandomTreeFormattedAsString(false, 2, 200, 50)
+                        .getBytes(StandardCharsets.UTF_8);
+            case LARGE:
+                return randomJsonGenerator.getRandomTreeFormattedAsString(true, 2, 2000, 400)
```

Comment:
why pretty-printing this one? and out of curiosity, why a seed of 2 for both of them?